index.js

```diff
@@ -0,0 +1,42 @@
+const testLineup = ['1B', '2B', '3B', 'C', 'OF', 'OF', 'OF', 'P', 'SS']
+
+function positionCount(lineup, testLineup) {
+  let positions = true
+  let positionsArr = lineup.map(position => position.position).sort()
+
+  if (positionsArr.length !== testLineup.length) return false
+
+  positionsArr.forEach((postition, index) => {
+    if (postition !== testLineup[index]) {
+      positions = false
+    }
+  })
+
+  return positions
+}
+
+function validateLineup(lineup) {
+  let salary = lineup.reduce((acc, player) => acc + player.salary, 0)
+  let length = true
+  let teamRule = true
+  let gameRule = true
+
+  for (let i = 0; i < lineup.length; i++) {
+    if (lineup.salary > 45000) {
+      salary = false
```

Comment:
changing salary to false... reusing variable :(

also you dont need to check the salary for each player here. as you already summed them all using reduce before. move this check outside forloop